drop rsvp and fs-extra dependencies

craigteegarden opened the pull request:
Remove RSVP dependency using same method as https://github.com/ember-bootstrap/ember-bootstrap/pull/2055

jelhan reviewed (approved):
Looks good to me. But I haven't verified locally that it works as expected. Did you verified locally that it's working?

We don't have any test coverage for the blueprints. We rely on manual testing for those changes.

craigteegarden commented:
After running into issues using the blueprint, even with the currently published version, I fixed the issue and added a couple tests for the blueprint itself. I then could run the blueprint and verified it still worked with the blueprint fs changes (also the preprocessor test should now cover that too).
